.all-contributorsrc

```diff
@@ -33,5 +33,5 @@
   "repoType": "github",
   "repoHost": "https://github.com",
   "projectName": "nx-release",
-  "projectOwner": "kreuzerk"
+  "projectOwner": "CarlosGilB"
```

Comment:
Project owner should not be changed, I guess that comes from cloning and running it on your side